code/modules/scripting/Implementations/_Logic.dm

```diff
@@ -151,159 +10,8 @@
-proc/n_str2num(var/string)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_str2num() called tick#: [world.time]")
-	if(istext(string))
-		return text2num(string)
-
-// Clamps N between min and max
-/proc/n_clamp(var/num, var/min = 0, var/max = 1)
-	if(isnum(num) && isnum(min) && isnum(max))
-		return Clamp(num, min, max)
-
-// Number shit
-proc/n_num2str(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_num2str() called tick#: [world.time]")
-	if(isnum(num))
-		return num2text(num)
-
-// Squareroot
-proc/n_sqrt(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_sqrt() called tick#: [world.time]")
-	if(isnum(num))
-		return sqrt(num)
-
-// Magnitude of num
-proc/n_abs(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_abs() called tick#: [world.time]")
-	if(isnum(num))
-		return abs(num)
-
-// Round down
-proc/n_floor(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_floor() called tick#: [world.time]")
-	if(isnum(num))
-		return Floor(num)
-
-// Round up
-proc/n_ceiling(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_ceil() called tick#: [world.time]")
-	if(isnum(num))
-		return Ceiling(num)
-
-// Round to nearest integer
-proc/n_round(var/num)
-	//writepanic("[__FILE__].[__LINE__] \\/proc/n_round() called tick#: [world.time]")
-	if(isnum(num))
-		if(num-round(num)<0.5)
-			return round(num)
-		return Ceiling(num)
-
-// END OF BY DONKIE :(
-
-/proc/n_sin(var/const/x)
-	if(isnum(x))
-		return sin(x)
-
-/proc/n_cos(var/const/x)
-	if(isnum(x))
-		return cos(x)
-
-/proc/n_tan(var/const/x)
-	if(isnum(x))
-		return Tan(x)
-
-/proc/n_csc(var/const/x)
-	if(isnum(x))
-		return Csc(x)
-
-/proc/n_cot(var/const/x)
-	if(isnum(x))
-		return Cot(x)
-
-/proc/n_sec(var/const/x)
-	if(isnum(x))
-		return Sec(x)
-
-/proc/n_asin(var/const/x)
-	if(isnum(x))
-		return arcsin(x)
-
-/proc/n_acos(var/const/x)
-	if(isnum(x))
-		return arccos(x)
-
-/proc/n_isInRange(var/const/x, var/const/min, var/const/max)
-	if(isnum(x) && isnum(min) && isnum(max))
-		return IsInRange(x, min, max)
-
-/proc/n_lcm(var/const/a, var/const/b)
-	if(isnum(a) && isnum(b))
-		return Lcm(a, b)
-
-/proc/n_root(var/const/n, var/const/x)
-	if(isnum(n) && isnum(x))
-		return Root(n, x)
-
-/proc/n_toDegrees(var/const/x)
-	if(isnum(x))
-		return ToDegrees(x)
-
-/proc/n_toRadians(var/const/x)
-	if(isnum(x))
-		return ToRadians(x)
-
-/proc/n_max(...)
-	return max(arglist(args))
-
-/proc/n_min(...)
-	return min(arglist(args))
-
-/proc/n_log(var/num)
-	if(isnum(num) && 0 < num)
-		return log(num)
-
-/proc/n_replacetext(text, r, with)
-	return replacetext(text, r, with)
-
-/proc/n_replacetextEx(text, r, with)
-	return replacetextEx(text, r, with)
-
-/proc/n_jointext(list, glue)
-	return jointext(list, glue)
-
-/proc/n_splittext(text, delim)
-	return splittext(text, delim)
\ No newline at end of file
+//Converts a list to a string
+/proc/list_implode(var/list/li, var/separator)
```

Comment:
Or this?

Reply:
this is only used in one place: https://github.com/ParadiseSS13/Paradise/blob/4c2532b9dbb2838c15c24ab31f1294d4841c2522/code/modules/nano/modules/ert_manager.dm#L69